adr/unity-folder-structure.md

```diff
@@ -0,0 +1,29 @@
+# Unity Folder Structure and File names
+
+Currently our Folder and File structure in our unity project is not consistent.
+There are Scripts in Scenes Folders, Scenes in no folder at all, etc. Also the Filenames do not follow a same Text Case.
+
+## Possible Solutions
+
+- Change nothing
+- Only put files in the correct folders, which are correctly named.
+  - for example put all World Scenes into the worlds folder
+  - Scripts only in the scripts folder, with subfolders which are named to display where the script belongs to
+    - also applies to all other stuff like prefabs, scenes, sprites, tiles, animations, ...
+  - no longer needed scenes/scripts/etc. should be moved to "old" folder in the respective parent Folder.
+    - or deleted?
```

Comment:
deleted.
Why should we keep outdated states?
That's what Git is for.

Reply:
yes because nobody said anything

Reply:
fixed now